content/documentations/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Documentations for Apache Camel
+---
+
+## User Manual
+
+The User Manual is a comprehensive guide meant to answer all your questions - from how to begin contributing to Apache Camel, to architecture and integration patterns.
+
+ - [How to Get Started](/manual/latest/getting-started.html)
+ - [Enterprise Integration Patterns](/manual/latest/enterprise-integration-patterns.html)
+ - [Camel 2.x to 3.0 Migration Guide](/manual/latest/camel-3-migration-guide.html)
+ - [Camel 3.x Upgrade Guide](/manual/latest/camel-3x-upgrade-guide.html)
+ - [FULL USER MANUAL](/manual/latest/)
+
+## Project Documentation
+
+Apache Camel encompasses several projects which offer many services to the user. Check out the manuals and documentation for these projects here :
+
+- [Camel-K](/camel-k/latest/)
+- [Camel Quarkus](/camel-quarkus/latest/)
+- [Camel Kafka Connector](/camel-kafka-connector/latest/)
+- [JavaDocs documentation](https://www.javadoc.io/doc/org.apache.camel/camel-api/latest/index.html) for Apache's APIs for Camel Core, Camel Catalog, Camel Cloud, Camel Cluster Support and Camel SPI etc.
```

Comment:
For text I'd go with something like:

```suggestion
- [API documentation](https://www.javadoc.io/doc/org.apache.camel/camel-api/latest/index.html)
```

Though this relates more to the Camel itself not to the sub-projects, so it needs be with in the user manual section. Perhaps that section needs to be renamed also as it won't be containing just the user manual.

Reply:
For this, should I move the API documentation to User Manual (and rename it to something else)? Or let it stay under sub-projects for now?

Reply:
The API documentation is for Camel not for the sub-projects, that's why I would not put the API documentation under sub-projects.